### ISSUE TITLE
Ratings cannot be submitted without JS

### ISSUE THREAD

**Issue by tofumatt** [mozilla/addons#10017]
### Describe the problem and steps to reproduce it:
Rating buttons submit to `<form action="">` when JS isn't available. See: https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/OverallRating.js#L102

This means the ratings can't be submitted without JS.

### What happened?
I clicked a button and the page submitted to itself, so it seemed like nothing happened.

### What did you expect to happen?
I click the button to rate the add-on and the add-on is shown as rated by me with the appropriate amount of stars.

### Anything else we should know?
I encountered this in development mode and maybe the buttons actually show content elsewhere... as none of my CSS displayed on the development server without JS, but that could be separate.

### Screenshots
<img width="316" alt="screenshot 2017-01-01 13 05 59" src="https://cloud.githubusercontent.com/assets/90871/21581288/87a9a82c-d023-11e6-9a6b-9ef89d3a2fa3.png">
